data/templates/high-availability/keepalived.conf.j2

```diff
@@ -5,6 +5,11 @@
 global_defs {
     dynamic_interfaces
     script_user root
+{% if vrrp.global_parameters is vyos_defined %}
```

Comment:
This line is superfluous an can be dropped

Reply:
done!